app/views/donors/_edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= form_for(@donor) do |f| %>
+
+<div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+    <%= link_to @donor do %>
+    <i class="fa fa-close fa-lg"></i>
+    <% end %>
+</div>
+
+<div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
```

Comment:
Bootstrap columns always need to be nested directly under a row, or the margins will be messed up. 😀